privacyideautils/yubikey.py

```diff
@@ -145,10 +145,13 @@ def enrollYubikey(digits=6, APPEND_CR=True, debug=False, unlock_key=None, access
     # Do the fixed string:
     if prefix_serial:
         Cfg.fixed_string(serial)
-    if fixed_string:
+    elif fixed_string:
```

Comment:
This changes the logic. 
Till now: the fixed_string would override the prefix_serial.
With your change: the prefix_serial would override the fixed_string.